frontend/src/components/common/Button/Button.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+
+interface Props {
+  title: string;
+  size: TSize;
+  disabled?: boolean;
+  onClick: () => void;
+}
+
+type TSize = 'md' | 'lg';
+type TSizeDetail = 'width' | 'height';
+
+const sizes: Record<TSize, Record<TSizeDetail, string>> = {
+  lg: {
+    width: '136px',
+    height: '36px',
+  },
+  md: {
+    width: '290px',
+    height: '42px',
+  },
+};
+
+function Button({ title, size, disabled, onClick }: Props) {
+  return (
+    <ButtonElement onClick={onClick} size={size} disabled={disabled}>
+      {title}
+    </ButtonElement>
+  );
+}
+
+export default Button;
+
+const ButtonElement = styled.button<Pick<Props, 'size'>>`
+  background: #2ac1bc;
```

Comment:
```suggestion
  background: ${({theme}) => theme.color.primary200}; 
```
이런 식으로 theme를 활용하는게 좋아보입니다.